index.html

```diff
@@ -39,22 +39,276 @@ <h1 class="main-text font-poppins">
           <ul>
             <li>
               <a href="https://twitter.com/_JavierAybar">
-                <img class="social-media-icon" src="images/Twitter-icon.svg" alt="Twitter" />
+                <img
+                  class="social-media-icon"
+                  src="images/Twitter-icon.svg"
+                  alt="Twitter"
+                />
               </a>
             </li>
             <li>
               <a href="https://www.linkedin.com/in/javier-aybar-932376274/">
-                <img class="social-media-icon" src="images/Linkedin icon.svg" alt="Linkedin" />
+                <img
+                  class="social-media-icon"
+                  src="images/Linkedin icon.svg"
+                  alt="Linkedin"
+                />
               </a>
             </li>
             <li>
               <a href="https://github.com/JavierAybar">
-                <img class="social-media-icon" src="images/Github-icon.svg" alt="GitHub" />
+                <img
+                  class="social-media-icon"
+                  src="images/Github-icon.svg"
+                  alt="GitHub"
+                />
               </a>
             </li>
           </ul>
         </section>
       </section>
     </main>
+    <!--Works-->
+    <section class="cards-container-flex">
+      <section class="cards">
```

Comment:
I don't see the need for two direct containers as both are wraping the same thing

Reply:
Hi! @souad988 I already made all the changes, except this one, the reason why I use another section(class="cards-container-flex") is because with this one I center the "works" section in the middle of the page. with the section(class="cards") I can't apply it because it has (flex:grid). 
In the 375px x 718px version I have no problems, but without changing the size a bit it looks like this
![127 0 0 1_5500_index html (1)](https://github.com/JavierAybar/Portfolio/assets/129561152/679370f5-3ee8-4638-908a-875cb8619a47)
in the 375px 718px don't have problems
![127 0 0 1_5500_index html](https://github.com/JavierAybar/Portfolio/assets/129561152/e15656bf-13ac-4e94-9884-37d73a58a1c1)